Lib/defconQt/glyphView.py

```diff
@@ -187,27 +292,38 @@ def __init__(self, glyph, parent=None):
         glyphMenu.addAction("&Jump", self.changeGlyph, "J")
         menuBar.addMenu(glyphMenu)
 
-        toolBar = QToolBar(self)
+
+        self._displaySettings = DisplayStyleSettings("&Display", self, self._displayChanged)
+        menuBar.addMenu(self._displaySettings.menuWidget)
+
+        self._toolBar = toolBar = QToolBar(self)
         toolBar.setMovable(False)
         toolBar.setContentsMargins(2, 0, 2, 0)
-        selectionToolButton = toolBar.addAction("Selection", self.view.setSceneSelection)
+        selectionToolButton = toolBar.addAction("Selection", self._redirect('view', 'setSceneSelection'))
         selectionToolButton.setCheckable(True)
         selectionToolButton.setChecked(True)
         selectionToolButton.setIcon(QIcon(":/resources/cursor.svg"))
-        penToolButton = toolBar.addAction("Pen", self.view.setSceneDrawing)
+        penToolButton = toolBar.addAction("Pen", self._redirect('view', 'setSceneDrawing'))
         penToolButton.setCheckable(True)
-        penToolButton.setIcon(QIcon(":/resources/curve.svg"))
-        rulerToolButton = toolBar.addAction("Ruler", self.view.setSceneRuler)
+        selectionToolButton.setIcon(QIcon(":/resources/cursor.svg"))
+        rulerToolButton = toolBar.addAction("Ruler", self._redirect('view', 'setSceneRuler'))
```

Comment:
When a glyph is changed I create a new GlyphView and throw away the old one. Stuff that I want to keep is handled by the Window. I do this, so I don't need to keep track of state changes that happen between glyph changes (like subscriptions to the glyph object), I think switching layers etc. whatnot is enough to manage in GlyphView

the switch is performed here: https://github.com/graphicore/trufont/commit/d2a3e6dcc58661ff6751af5f9906a27c9df4f5e9#diff-4afada8b45ba97a28f2ff0c5a9995dc0R351



That's what I meant with:

> I changed some things regarding object hierarchy. MainGfxWindow now acts much more as a controller for GlyphView.

The redirect thing keeps me from having to rewire the signals after I replaced the view.